lib/posix/epoll.nim

```diff
@@ -35,18 +35,13 @@ const
   EPOLL_CTL_MOD* = 3          # Change file descriptor epoll_event structure.
 
 type
-  epoll_data* {.importc: "union epoll_data",
+  EpollData* {.importc: "union epoll_data",
       header: "<sys/epoll.h>", pure, final.} = object # TODO: This is actually a union.
-    #thePtr* {.importc: "ptr".}: pointer
-    fd* {.importc: "fd".}: cint
-    when defined(linux) and defined(amd64):
-      u32: uint32 # this field ensures that binary size is right - it cannot be
-                  # used because its offset is wrong
-    #u64*: uint64
+    u64* {.importc: "u64".}: uint64
 
-  epoll_event* {.importc: "struct epoll_event", header: "<sys/epoll.h>", pure, final.} = object
+  EpollEvent* {.importc: "struct epoll_event", header: "<sys/epoll.h>", pure, final.} = object
```

Comment:
what was the point to rename this type?

Reply:
Convention

Reply:
To elaborate a bit, I basically found ``epoll_event()`` used as a constructor sorta strange looking. So I decided to rename it so that it looks like a proper type.